code/MessageQueue.php

```diff
@@ -29,6 +29,10 @@ class MessageQueue
             )
         )
     );
+	
+    /**Optionally defined to 'true' via YML, this will stop MessageQueue_Process 
```

Comment:
Please cleanup. `/**` and `*/` should be on their own lines, and each other line should have a leading `*`